spec/xclarity_client_node_spec.rb

```diff
@@ -2,44 +2,45 @@
 
 describe XClarityClient do
   before :all do
-    WebMock.allow_net_connect! # -- This line should be uncommented if you're using external mock test
+    # -- The next line should be uncommented
+    # if you're using external mock test
+    WebMock.allow_net_connect!
 
     conf = XClarityClient::Configuration.new(
       username:   ENV['LXCA_USERNAME'],
       password:   ENV['LXCA_PASSWORD'],
-      host:       ENV['LXCA_HOST'],
+      host:       'example.com',
```

Comment:
I changed it to test that 'https://' was prepended properly, but yes it should be changed back.  I will update that as well.